libs/pdi-scanner/examples/get_test_string.rs

```diff
@@ -57,7 +58,7 @@ async fn main() -> color_eyre::Result<()> {
     for n in 1..=config.times {
         println!(
             "{n:02}: get_test_string result: {:?}",
-            client.get_test_string(Duration::from_secs(1))
+            timeout(Duration::from_secs(1), client.get_test_string()).await
```

Comment:
This is an example of one of the benefits of moving to async: we no longer need to take a `Duration` parameter on each of the methods that expect a response. Instead, we can just use `tokio`’s normal timeout mechanism.